rollup.config.js

```diff
@@ -60,14 +60,22 @@ export default [{
     footer: `\`
   if (window.profilerEnabled) window.tag.text = window.tag.text.replace('let profilerEnabled = false;', '\$&\\nstartProfiler();')
   document.children[0].append(window.tag)
-  const port = chrome.runtime.connect()
-  port.onMessage.addListener(window.postMessage.bind(window))
+  const sendMessage = chrome.runtime.sendMessage
+  const postMessage = window.postMessage.bind(window)
+  chrome.runtime.onMessage.addListener((message, sender) => {
+    const messageIsFromBackgroundPage = sender && sender.id === chrome.runtime.id
+    if (!messageIsFromBackgroundPage) {
```

Comment:
Unnecessarily long variable name. Please either include the condition in the if statement and use a comment or shorten the variable name to something more readable.

Reply:
Changed `messageIsFromBackgroundPage` to `fromBackground`. Is it OK now?